### PR TITLE
Fixed issue #375 LinkInspector dropdown in evolving relationship

### PR DESCRIPTION
**Pull request by Domily11**
Evolving relationships that start with no relationship can choose second option now

### CODE REVIEW

File: leaf-ui/rappid-extensions/LinkInspector.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ var LinkInspector = Backbone.View.extend({
         } else {
             this.$el.html(_.template(this.template)());
             $('#constant-links').val(values[0].trim());
-            this.evolvingRelations = false;
+            this.evolvingRelations = false; //redundant????
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/LinkInspector.js]: Remove comment from your commit but definitely note this somewhere (maybe in a google doc?)

File: leaf-ui/rappid-extensions/LinkInspector.js [hunk 2]
```diff
@@ -278,7 +275,7 @@ var LinkInspector = Backbone.View.extend({
             '.marker-source': {'d': '0'},
             '.marker-target': {stroke: '#000000', "d": 'M 10 0 L 0 5 L 10 10 L 0 5 L 10 10 L 0 5 L 10 5 L 0 5'}
         });
-        this.cell.label(0, {position: 0.5, attrs: {text: {text: begin + " | " + end}}});
+        this.cell.label(0, {position: 0.5, attrs: {text: {text: begin + " | " + end}}}); //do we need the codes then in lines 239 and 245????
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/LinkInspector.js]: Remove this comment as well, but still note it somewhere

File: leaf-ui/rappid-extensions/LinkInspector.js [hunk 3]
```diff
@@ -370,7 +367,7 @@ var LinkInspector = Backbone.View.extend({
         if (selector == "link-type-end") {
             var dupVal = $('#link-type-begin').val();
             $("select#link-type-end option").filter("[value='" + dupVal + "']").remove();
-        }
+        }//feels very inefficient????
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/LinkInspector.js]: Remove comment

File: leaf-ui/rappid-extensions/LinkInspector.js [hunk 4]
```diff
@@ -229,9 +227,8 @@ var LinkInspector = Backbone.View.extend({
         //Enable the end select
         if (begin == "no") {
             this.setSelectValues('#link-type-end', 'Evolving');
-            $("#link-type-end").prop('disabled', true);
-            $("#link-type-end").css("background-color", "gray");
-            $("#repeat-error").text("");
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/LinkInspector.js:234]: Why was this line deleted?

**Review by leshghi — APPROVED** [review]
This looks great to me - thanks for the fix! I made a few notes about removing comments from code and a quick question about a removed line so just respond/make those changes and then you should be all set!

File: leaf-ui/rappid-extensions/LinkInspector.js (hunk 4 above)
Comment (Domily11) [reply leaf-ui/rappid-extensions/LinkInspector.js:234]: I didn't though? I just changed the true to false and the gray to nothing. I'm not sure if we wanted the Saved! to appear or not when the second option was chosen. I assumed yes, so I took line 234 out because otherwise it wasn't appearing
